authors/apps/usernotifications/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.urls import path
+from . import views
+
+app_name = 'notifications'
+
+urlpatterns = [
+    path(
+        'notifications/unsubscribe/<token>',
+        views.SubscribeUnsubscribeAPIView.as_view(),
+        name="email-unsubscribe"
+        ),
+    path(
+        'notifications/',
+        views.AllNotificationsAPIview.as_view(),
+        name="all-notifications"
+        ),
+    path(
+        'notifications/subscribe/',
```

Comment:
I think it would be better if you used one endpoint for both Subscribe and UnSubscribe.